pkg/server/netfilterrules.go

```diff
@@ -1293,12 +1298,30 @@ func (n *nftState) addIPRule(addrs []string, chain *nftables.Chain, policyName s
 				SetName:        ipSet.Name,
 				SetID:          ipSet.ID,
 			},
+			&expr.Meta{Key: expr.MetaKeyMARK, Register: 0x1},
+			&expr.Bitwise{
+				SourceRegister: 0x1,
+				DestRegister:   0x1,
+				Len:            4,
+				Mask:           binaryutil.NativeEndian.PutUint32(^uint32(0x20000)), // 0xfffdffff
+				Xor:            binaryutil.NativeEndian.PutUint32(0),
```

Comment:
should we put uint32 also around the constants here? :)
Maybe we should add them as constantst ^^